src/core/account/account-init.js

```diff
@@ -91,7 +91,7 @@ export async function makeAccount(
   appId: string,
   loginTree: LoginTree,
   loginType: LoginType,
-  opts: EdgeAccountOptions = {}
+  opts: EdgeAccountOptions
```

Comment:
I'm guessing that it's not optional anymore since all the places it's called already has a default value?

Reply:
It's still optional, because the parameter list has it as `opts = {}` on all the `EdgeContext` methods. The difference is that passing `null` is no longer valid. By the time we get into the heart of the core, though, it's no longer optional because the `opts = {}` bit ensures it already exists.